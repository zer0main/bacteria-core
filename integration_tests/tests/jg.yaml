board_width: 15
board_height: 15
moves_number: 2
units:
    - coordinates: [10, 10]
      mass: 15
      direction: 3
      team: 0
      instruction: 0
