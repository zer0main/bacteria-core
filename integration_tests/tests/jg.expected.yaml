board_width: 15
board_height: 15
units:
    - coordinates: [10, 10]
      mass: 6
      direction: 3
      team: 0
      instruction: 2

    - coordinates: [10, 9]
      mass: 3
      direction: [0, 3]
      team: 0
      instruction: 2
